okio/src/main/java/okio/BufferedSource.java

```diff
@@ -533,6 +541,27 @@
   boolean rangeEquals(long offset, ByteString bytes, int bytesOffset, int byteCount)
       throws IOException;
 
+  /**
+   * Returns a new `BufferedSource` that can read data from this `BufferedSource` without consuming
```

Comment:
These docs are still with kdoc formatting, is that intentional?

Reply:
Fixed!